release.sh

```diff
@@ -10,5 +10,9 @@ cd $REPO_ROOT_DIR
 ./build.sh
 ./gradlew publish
 
+# Scripts below expects S3_BUCKET and SDK_VERSION env vars. To be supplied by Jenkins.
```

Comment:
Yes. They are passed in from the Build job. And, like you noticed they have sane defaults.